src/xenialdan/PocketRadio/task/SongPlayerTask.php

```diff
@@ -18,31 +18,29 @@ class SongPlayerTask extends Task
 {
     public $song = null;
     public $songfilename = "";
-    /** @var Plugin|Loader */
+    /** @var PluginBase */
```

Comment:
???????????????????????????????????????????????

Reply:
This change is not related to the main objective of this PR, I will remove it.